src/components/switch/switch.tsx

```diff
@@ -10,7 +10,7 @@ const rootClassNames = classNames(
 );
 
 const thumbClassNames = classNames(
-    'block h-4 w-4 rounded-full bg-neutral-300 transition-transform duration-100 will-change-transform', // Default
+    'block size-4 rounded-full bg-neutral-300 transition-transform duration-100 will-change-transform', // Default
```

Comment:
Missed this when building the component, but can you add `md:text-base/tight` to the `labelClassNames`, please?